R/calculateDMM.R

```diff
@@ -137,8 +142,9 @@ setMethod("calculateDMN", signature = c(x = "ANY"), .calculate_DMN)
 #' @rdname calculateDMN
 #' @export
 setMethod("calculateDMN", signature = c(x = "SummarizedExperiment"),
-    function(x, abund_values = exprs_values, exprs_values = "counts", transposed = FALSE, ...){
-        mat <- assay(x, abund_values)
+    function(x, assay_name = abund_values, abund_values = exprs_values, exprs_values = "counts", 
+             transposed = FALSE, ...){
+        mat <- assay(x, assay_name)
```

Comment:
Is there a deprecation msg for exprs_values and abund_values? Would be good if not yet.

Reply:
Not yet, I can add

Reply:
Actually it is not possible to detect whether user has used assay_name, abund_values, or exprs_values when it is implemented like this

Reply:
Hmm right.. if there is a fluent way of doing this I think it would be worth it. 

However "abund_values" or "exprs_values" might be something that is used also in SCE tools, we might like to allow (not deprecate) synonyms that are used in other frameworks.

Reply:
Yes.